xml/containers-support.xml

```diff
@@ -349,7 +349,7 @@
   </itemizedlist>
 
   <para>
-   Container images are labeled as Tech Preview are marked as such at
+   Container images are labeled as Tech Preview and are marked as such at
```

Comment:
```suggestion
   Container images are labeled as <literal>Tech Preview</literal> and are marked as such at
```

Reply:
OK, I noticed that the rest of the para uses the correct terminology ('technology preview'). I assume 'Tech Preview' is the label being used in registry.suse.com, therefore I would put a ```<literal/>``` element around it.